include/msg.h

```diff
@@ -23,117 +23,124 @@ typedef int msg_rid_t;
  * Message types
  */
 
+/* clang-format off */
 
 enum {
 	/* File operations */
 	mtOpen = 0, mtClose, mtRead, mtWrite, mtTruncate, mtDevCtl,
 
 	/* Object operations */
-	mtCreate, mtDestroy, mtSetAttr, mtGetAttr,
+	mtCreate, mtDestroy, mtSetAttr, mtGetAttr, mtGetAttrAll,
 
 	/* Directory operations */
 	mtLookup, mtLink, mtUnlink, mtReaddir,
 
 	mtCount
 };
 
+/* clang-format on */
+
 
 #pragma pack(push, 8)
 
 
+struct _attr {
+	long long val;
+	int err;
+};
+
+
+struct _attrs {
```

Comment:
Just one char difference between names, easy to make a mistake or misread the code. Perhaps `_attrArray`, `_attrAll` or `_attrTab`?